client/src/components/Slider/Slider.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
```

Comment:
These files for the Slider component are part of a separate PR - can you remove them from this PR?